Scoop/Controller/PostRideLocationViewController.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  PostRideLocationViewController.swift
+//  Scoop
+//
+//  Created by Elvis Marcelo on 3/9/22.
+//
+
+import UIKit
+
+class PostRideLocationViewController: UIViewController {
+    
+    private var containerView = UIView()
+    
+    private let transportationMethodLabel = UILabel()
+    private let transportationTextField = UITextField()
+    
+    private let arrivalTextField = UITextField()
+    private let departureTextField = UITextField()
+    private let locationLabel = UILabel()
+
+    private let fieldSpace = 40
+    private let labelSpace = 5
+    private let textFieldSpace = 20
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        navigationItem.title = "Trip Details"
+        view.backgroundColor = .white
+        
+        setupContainerView()
+        setupTransporationMethod()
+        setupLocation()
+    }
+    
+    func setupContainerView() {
+        containerView.backgroundColor = .clear
+        view.addSubview(containerView)
+        
+        containerView.snp.makeConstraints { make in
+            make.centerY.equalToSuperview()
+            make.leading.equalToSuperview().inset(20)
+            make.trailing.equalToSuperview().inset(20)
+        }
+    }
+    
+    func setupTransporationMethod() {
+        transportationMethodLabel.text = "Method of Transportation"
+        transportationMethodLabel.textColor = .black
+        containerView.addSubview(transportationMethodLabel)
+        
+        transportationMethodLabel.snp.makeConstraints { make in
+            make.top.equalToSuperview().offset(fieldSpace)
+            make.centerX.equalToSuperview()
+        }
+        
+        transportationTextField.placeholder = "select"
+        transportationTextField.textColor = .black
+        containerView.addSubview(transportationTextField)
+        
+        transportationTextField.snp.makeConstraints { make in
+            make.leading.equalTo(transportationMethodLabel.snp.leading)
```

Comment:
change to `make.leading.equalTo(transportationMethodLabel)`

Whenever you are setting an anchor realtive the same anchor in another view, you dont need to include the `snp.anchor` part